src/components/CardJob.jsx

```diff
@@ -0,0 +1,53 @@
+import { baseUrl } from '../constants';
+import './CardJob.css';
+import { ButtonLanguage } from './ButtonsLanguages';
+
+function CardJob(props) {
+  const {
+    company,
+    logo,
+    position,
+    postedAt,
+    contract,
+    location,
+    featured,
+    languages,
+    role,
+    level,
+  } = props;
+
+  const logoBase = baseUrl + logo.slice(1);
+
+  return (
+    <div className="card">
+      <img className="img-logo" src={logoBase} alt="logo" />
+      <div className="first-line">
+        <p className="company">{company}</p>
+        {props.new ? <p className="new">NEW!</p> : ''}
```

Comment:
You used destructuring for all other props but not this one. Why?

Reply:
Because  world 'new' can not be used :)

Reply:
This is funny :)
But you always can rename it.
const { new: isNew } = props